src/dev/i18n/utils.js

```diff
@@ -84,3 +85,26 @@ export function* traverseNodes(nodes) {
     }
   }
 }
+
+/**
+ * @param {string} content
```

Comment:
__nit:__ please add a detailed description here so that it is clear what this function does